diff -ur a/src/plugins/cpptools/headerpathfilter.cpp b/src/plugins/cpptools/headerpathfilter.cpp
--- a/src/plugins/cpptools/headerpathfilter.cpp	2021-01-18 21:51:51.689284592 +0000
+++ b/src/plugins/cpptools/headerpathfilter.cpp	2021-01-18 21:51:31.236921330 +0000
@@ -51,8 +51,10 @@
     for (const HeaderPath &headerPath : headerPaths)
         filterHeaderPath(headerPath);
 
+#if 0 // broken and unnecessary
     if (useTweakedHeaderPaths == UseTweakedHeaderPaths::Yes)
         tweakHeaderPaths();
+#endif
 }
 
 bool HeaderPathFilter::isProjectHeaderPath(const QString &path) const
