diff --git a/usbmuxd2/sysconf/sysconf.cpp b/usbmuxd2/sysconf/sysconf.cpp
index 07d52f0..098393c 100644
--- a/usbmuxd2/sysconf/sysconf.cpp
+++ b/usbmuxd2/sysconf/sysconf.cpp
@@ -371,6 +371,7 @@ void sysconf_fix_permissions(int uid, int gid){
 
 
 bool sysconf_try_getconfig_bool(std::string key, bool defaultValue){
+    return defaultValue;
     plist_t p_boolVal = NULL;
     cleanup([&]{
         safeFreeCustom(p_boolVal, plist_free);
